app.go

```diff
@@ -6,7 +6,8 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/hacksoc-manchester/www/handlers"
+	_ "github.com/alexdmtr/www/config"
+	"github.com/alexdmtr/www/handlers"
```

Comment:
These paths must be updated back to "github.com/hacksoc-manchester/www/".

Reply:
Agreed